swagger.json

```diff
@@ -4194,6 +4194,12 @@
                     "description": "The address of the job destination, as it would be recognized if provided to maps.google.com",
                     "example": "123 Main St, Philadelphia, PA 19106"
                 },
+                "destination_address_id": {
+                    "type": "integer",
+                    "format": "int64",
+                    "description": "ID of the job destination associated with an address book entry.",
```

Comment:
nit: add "The" at the beginning of this description for consistency? We have things like `"description": "The ID of this tag."` in other places

Reply:
Good point. I'm inclined to keep it as is because I'm seeing it done both ways in our docs (lol), but also because most of the dispatch/driver docs api docs start with `ID of ...`

Reply:
seems fine to me then